cephfs/admin/workflow_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
-	"path"
+	pathpkg "path"
```

Comment:
Why?

Reply:
Beacause "path" was used elsewhere in the file for actual path values. So "path" the package was called "path package" abbreviated to "pathpkg" :-)